models/glacier/avalanche_c/glacier_v2_avalanche_c_trades.sql

```diff
@@ -0,0 +1,102 @@
+{{ config(tags=['dunesql'],
+    alias = alias('trades')
+    ,partition_by = ['block_month']
+    ,materialized = 'incremental'
+    ,file_format = 'delta'
+    ,incremental_strategy = 'merge'
+    ,unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+    ,post_hook='{{ expose_spells(\'["avalanche_c"]\',
```

Comment:
can you remove this post-hook?
we don't need to expose this spell, as the intention is to push queries to `dex.trades` instead